core/src/trezor/wire/message_handler.py

```diff
@@ -38,14 +43,33 @@ def wrap_protobuf_load(
             )
         return msg
     except Exception as e:
-        if __debug__:
+        if __debug__ and utils.ALLOW_DEBUG_MESSAGES:
             log.exception(__name__, e)
         if e.args:
             raise DataError("Failed to decode message: " + " ".join(e.args))
         else:
             raise DataError("Failed to decode message")
 
 
+if utils.USE_THP:
+    from trezor.enums import ThpMessageType
+
+    def get_msg_name(msg_type: int) -> str | None:
```

Comment:
We should probably extend rust protobuf parser to handle this, can help with that.